### PR TITLE
JWT Validator and API Key Authenticator improvements

### PR DESCRIPTION
**Pull request by harsha89**

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/jwt/RevokedJWTTokensRetriever.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public void run() {
 
     private void loadRevokedJWTTokensFromWebService() {
 
-        RevokedJWTTokenDTO[] revokedJWTTokenDTOS = retrieveRevokedJWTTokensData();
+        RevokedJWTTokenDTO[] revokedJWTTokenDTOS     = retrieveRevokedJWTTokensData();
```
Comment (mushthaq33) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/jwt/RevokedJWTTokensRetriever.java]: Seems like an extra space is added.

File: components/apimgt/org.wso2.carbon.apimgt.jms.listener/src/main/java/org/wso2/carbon/apimgt/jms/listener/utils/JMSMessageListener.java [hunk 2]
```diff
@@ -362,6 +358,10 @@ private void handleRevokedTokenMessage(String revokedToken, long expiryTime) {
             }
             Utils.removeCacheEntryFromGatewayCache(revokedToken);
             Utils.putInvalidTokenEntryIntoInvalidTokenCache(revokedToken, cachedTenantDomain);
+            //Clear the API Key cache if revoked token is in the JWT format
+            if(isJwtToken) {
```
Comment (mushthaq33) [inline components/apimgt/org.wso2.carbon.apimgt.jms.listener/src/main/java/org/wso2/carbon/apimgt/jms/listener/utils/JMSMessageListener.java]: There is a formatting issue

Comment (harsha89) [reply components/apimgt/org.wso2.carbon.apimgt.jms.listener/src/main/java/org/wso2/carbon/apimgt/jms/listener/utils/JMSMessageListener.java]: Addressed

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/jwt/RevokedJWTTokensRetriever.java (hunk 1 above)
Comment (harsha89) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/jwt/RevokedJWTTokensRetriever.java]: Addressed

**Review by mushthaq33 — APPROVED** [review]

**Pull request merged**